docs/tutorials/hera_sim_defaults.ipynb

```diff
@@ -18,36 +18,15 @@
    "cell_type": "code",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    sim1.run_sim(**sim1params)
</pre><p>This should be moved between line 3 and line 4. I'm pretty sure the main reason the two plots look different is that the random seed is different.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera_sim/pull/238/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>